packages/makeup-menu-button/dist/cjs/index.js

```diff
@@ -109,8 +115,12 @@ function _onMenuKeyDown(e) {
   }
 }
 function _onMenuItemSelect(e) {
-  if (this._buttonPrefix && e.detail.el.getAttribute("role") === "menuitemradio") {
-    this._buttonTextEl.innerText = "".concat(this._buttonPrefix, " ").concat(e.detail.el.innerText);
+  if (e.detail.el.getAttribute("role") === "menuitemradio") {
```

Comment:
To make this more readable/maintainable, it's better to avoid nested conditionals. In this case, if you're looking for a specific type of `role`, just exit fast if it's the wrong role...

```js
if (e.detail.el.getAttribute("role") !== "menuitemradio") {
  return;
}
// then just continue with the rest of the logic in the current inner conditional. 
```

Reply:
Part of dist folder.